README.md

```diff
@@ -1,6 +1,6 @@
 # Handbook
 
-This repository contains FLINT Handbook
+This repository contains [FLINT Handbook](https://moja-global.github.io/Handbook/)  
```

Comment:
Can you rephrase this along these lines: https://github.com/moja-global/community-website#introduction

Reply:
Yes surely